### PR TITLE
usb:remove usb_smartcard and usb_redir test cases on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:1967816
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -390,6 +390,7 @@
                     # Please configure temp repository url for package usbredir-server
                     # temprepo_url = "http://xxx"
         - usb_smartcard_sharing:
+            no ppc64 ppc64le
```
Comment (yanan-fu) [inline qemu/tests/cfg/usb.cfg:393]: `usb_redir` use `spicevmc` too, i guess it can be disabled for ppc too ?

**Comment by MiriamDeng** [comment]
# git am 2849.patch
Applying: usb:remove smart card test case since it's not supported on ppc

File: qemu/tests/cfg/usb.cfg (hunk 1 above)
Comment (MiriamDeng) [reply qemu/tests/cfg/usb.cfg:393]: Thanks for the reminder and they should be disabled too.

**Comment by MiriamDeng** [comment]
@yanan-fu could you please help to review it if you are free, thanks a lot.

**Comment by MiriamDeng** [comment]
Applying this patch on local
python3 ConfigTest.py  ... category=usb_device --dryrun and no more smart card and usb_redir test cases any more.
 (131/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.usb.usb_host.usb_one_time.usb_check_isobufs.with_hostbus_hostaddr.without_usb_hub.qemu-xhci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (132/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.usb.usb_multi_disk.one_disk_repeat.without_usb_hub.nec-xhci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (133/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.usb.usb_multi_disk.one_disk_repeat.without_usb_hub.qemu-xhci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (134/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.usb.usb_multi_disk.max_disk.without_usb_hub.nec-xhci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (135/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.usb.usb_multi_disk.max_disk.without_usb_hub.qemu-xhci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (136/136) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.qemu_img_check: CANCEL: Test cancelled due to --dry-run (0.00 s)

**Review by yanan-fu — APPROVED** [review]
Acked-by: Yanan Fu <yfu@redhat.com>

**Pull request merged**